codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -277,6 +278,27 @@ public boolean visit(Http2Stream stream) {
         }
     }
 
+    @Override
+    public void headersSent(int streamId, boolean sent) {
+        Http2Stream stream = stream(streamId);
+        if (stream == null) {
+            // TODO: Throw something?
```

Comment:
Anyone have a good suggestion on how I should handle this case? I believe this will never happen as the stream id has already been checked in the encoder before calling this mehod